packages/roosterjs-content-model-dom/lib/modelApi/editing/applyTableFormat.ts

```diff
@@ -261,6 +261,7 @@ export function setFirstColumnFormatBorders(
 
                 switch (rowIndex) {
                     case 0:
+                        cell.isHeader = false;
```

Comment:
Yes, the idea is that First Column does not apply to the first cell, at least that is the behaviour we implemented and I am just restoring it.

However I noticed that this can conflict with hasHeaderRow, so I made a change to apply the header status accordingly.

`cell.isHeader = !!format.hasHeaderRow;`